tethysapp/app_store/templates/app_store/home.html

```diff
@@ -104,9 +110,9 @@
 
 {% block scripts %}
 {{ block.super }}
-<script src="https://unpkg.com/bootstrap-table@1.16.0/dist/bootstrap-table.min.js"></script>
-<script src="{% static 'tethys_apps/vendor/imagesloaded.min.js' %}" type="text/javascript"></script>
-<script src="{% static 'tethys_apps/vendor/masonry.min.js' %}" type="text/javascript"></script>
+<script src="https://unpkg.com/bootstrap-table@1.20.2/dist/bootstrap-table.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.min.js" type="text/javascript"></script>
+<script src="{% static 'app_store/vendor/masonry.min.js' %}" type="text/javascript"></script>
 <script src="{% static 'tethys_apps/js/app_library.js' %}" type="text/javascript"></script>
```

Comment:
It seems to be working.  I did not test submitting a git repo, as I understand that would send an e-mail to Rohit.  But I was able to install an app, bring up the github repo (not all apps have a linked repo- I think it should be dimmed or hidden if that is the case, but Tethys3 app store did not), upgrade an installed app, and uninstall an installed app.  I did not try any installed apps, since we knew they wouldn't work...